File: tests/fixtures/code/new.py
"""Sample Python module - Updated."""

def greet(name, greeting="Hello"):
    """Greet a person with custom greeting."""
    return f"{greeting}, {name}!"


def farewell(name):
    """Say goodbye."""
    return f"Goodbye, {name}!"


class Calculator:
    """A simple calculator with more operations."""
    
    def add(self, a, b):
        return a + b
    
    def subtract(self, a, b):
        return a - b
    
    def multiply(self, a, b):
        return a * b
    
    def divide(self, a, b):
        if b == 0:
            raise ValueError("Cannot divide by zero")
        return a / b


if __name__ == "__main__":
    print(greet("World", "Hi"))
    print(farewell("World"))

File: tests/fixtures/code/new.js
// Utility functions - Updated

function add(a, b) {
    return a + b;
}

function subtract(a, b) {
    return a - b;
}

function multiply(a, b) {
    return a * b;
}

function divide(a, b) {
    if (b === 0) throw new Error('Cannot divide by zero');
    return a / b;
}

const config = {
    apiUrl: 'https://api.example.com',
    timeout: 10000,
    retries: 3
};

module.exports = { add, subtract, multiply, divide, config };
